scripts/variantstore/README.md

```diff
@@ -16,9 +39,9 @@ After using the json file to auth as that service account, you can query the
     bq --project_id <project> query --use_legacy_sql=false 'select sum(total_bytes_billed) as total from `region-us`.INFORMATION_SCHEMA.JOBS_BY_USER where creation_time > "2021-04-16 00:00:00" and creation_time < "2021-04-18 00:00:00" and total_bytes_billed > 1 limit 100'
```

Comment:
maybe put these queries in triple single quotes (```) to format as a code block

Reply:
I couldn't get this to work, and it seems to already be formatting this as a code block (this existed before this PR).